tensorflow_blade/workspace0.bzl

```diff
@@ -81,6 +81,17 @@ def _tf_blade_repositories():
         ],
     )
 
+    maybe_http_archive(
```

Comment:
since google-test is added as a `http_archive`, can we remove the google-test submodule?

Reply:
you mean `torch_blade/third_party/googletest` ?